doc/api/url.md

```diff
@@ -305,6 +305,31 @@ to percent-encode may vary somewhat from what the [`url.parse()`][] and
 
 Gets and sets the port portion of the URL.
 
+The port value may be a number or a string containing a number in the range
```

Comment:
Nit: remove `containing a number`

Reply:
@BridgeAR I disagree, I don't think it's clear enough without that. A string cannot be in a range of numbers.

Reply:
Yeah, you are probably right.